transcendence-app/test/auth/auth.e2e-spec.ts

```diff
@@ -25,19 +30,44 @@ describe('Auth Controller', () => {
       controllers: [AuthController],
     }).compile();
 
-    config = module.get<ConfigService<OAuth42Config>>(ConfigService);
-    app = module.createNestApplication();
+    app = moduleFixture.createNestApplication();
+    config =
+      moduleFixture.get<ConfigService<EnvironmentVariables>>(ConfigService);
+
+    const MemcachedStore = ConnectMemcached(session);
+    const memcachedHost = config.get('MEMCACHED_HOST') as string;
+    const memcachedPort = config.get('MEMCACHED_PORT') as string;
+
+    app.use(
+      session({
+        secret: config.get('SESSION_SECRET') as string,
+        resave: false,
+        saveUninitialized: false,
+        store: new MemcachedStore({
+          hosts: [`${memcachedHost}:${memcachedPort}`],
+          secret: config.get('MEMCACHED_SECRET'),
+        }),
+      }),
+    );
+    app.use(passport.initialize());
+    app.use(passport.session());
     app.init();
   });
 
-  it('should redirect to 42 oauth page', async () => {
+  it('Login [GET /login]', async () => {
```

Comment:
I'm not sure that the description string you have chosen is more descriptive that the string it used to have.

Reply:
Fixed, thanks